examples/ma_collections_example.py

```diff
@@ -0,0 +1,35 @@
+import asyncio
+from examples.common import Example, Config
+from flow_py_sdk import flow_client
+
+# -------------------------------------------------------------------------
+# Retrieve a collection by ID Class
+# -------------------------------------------------------------------------
+class GetCollectioByIdExample(Example):
+    def __init__(self) -> None:
+        super().__init__(tag="CL.1.", name="GetCollectioByIdExample", sort_order=201)
+    async def run(self, ctx: Config):
+        collection_id = '48338b132f33a7d8776adc777730c601e1d4f4ccb04261c066aad78269b14e9e'
```

Comment:
Assuming this collection exists is a bit fragile. Consider getting the latest/first block and a collection id from that block, then getting that collection.

Reply:
yes. done!